cosmos/airflow/graph.py

```diff
@@ -214,7 +218,7 @@ def create_task_metadata(
         }
 
         if test_behavior == TestBehavior.BUILD and node.resource_type in SUPPORTED_BUILD_RESOURCES:
-            task_id = f"{node.name}_{node.resource_type.value}_build"
+            task_id, args = _get_task_id_and_args(node, args, use_task_group, normalize_task_id, "build", True)
```

Comment:
```suggestion
            task_id, args = _get_task_id_and_args(node, args, use_task_group, normalize_task_id, "build", include_resource_type=True)
```
since it's a keyword arg, good to reference the arg while passing the value.

Reply:
Fixed in: ec13fdbeb4d8eee15491c811072763c8ca75fe17